src/app/highlight-cards/HighlightCard.ts

```diff
@@ -58,14 +59,21 @@ export const SFAPIHighlightCardToHighlightCard = (experienceUriPrefix: string, b
 
   const backgroundImageUrl = backgroundImage(sfApiHighlightCard, donateUriPrefix);
 
+  let href = replaceURLOrigin(experienceUriPrefix, blogUriPrefix, donateUriPrefix, sfApiHighlightCard.button.href);
+
+  // temp fix for 2024. Will need to think of something better and probably move this logic to SF for next year.
+  if (href.pathname.includes('christmas-challenge')) {
+    href = new URL(donateUriPrefix + '/christmas-challenge-2024');
```

Comment:
Hmm yeah for some reason even though I wrote this part second I didn't think of using the campaign family as a test. I guess just because I'm changing the URL I was focused just on the URL.

Reply:
and yeah I think this is all going to be substantially re-worked next time we edit it.

Reply:
Yeah, I don't think the other way is meaningfully better when we have to hardcode the result anyway.